src/main/PileupTrack.js

```diff
@@ -293,6 +293,7 @@ class PileupTrack extends React.Component {
     var trackingCtx = new dataCanvas.ClickTrackingContext(ctx, x, y);
     this.renderScene(trackingCtx);
     var vRead = trackingCtx.hit && trackingCtx.hit[0];
+    var alert = window.alert | console.log;
```

Comment:
I think you mean `||` here (surprised flow doesn't complain about this, actually)